react/modules/module05/README.md

```diff
@@ -1,47 +1,51 @@
+4) The customer has changed the requirements for the output of parts: 
+    - Now, when you click on each item from the search results, you do not need to open a modal, but open the information on a separate page. 
+    - Also add a "Back" link on this page. 
+    - Display the current position in the header. 
+    - The detail page should take data from the global state.
+    - If there is no data, then redirect to the home page.
+### Criteria for evaluation
 
-Задание будет оцениваться ментором в течении первых 6 недель. Для удобства проверки создайте Pull Request в ветку предыдущего задания(**МЕРЖИТЬ НЕ НАДО**).
+The task will be checked by the mentor during the first 6 weeks. Create a Pull Request to the branch of the previous task, for mentor's checking (**DO NOT MERGE**).
 
 
-1) Custom State Management - **9 баллов**
-2) Сортировка и пагинация  - **3 балла**
-3) Детали поиска в отдельной странице - **3 балла**
+1) Custom State Management - **9 points**
+2) Sorting and pagination - **3 points**
+3) Search details in a separate page - **3 points**
 
 
-Если хоть один из пунктов не выполнен, то задание оценивается в **0 баллов**.
+If at least one of the requirements is not completed, then the task is evaluated as **0 points**.
 
-### Требования к репозиторию
+### Repository requirements
 
-- задание выполняется в **приватном репозитории школы** [Как работать с приватным репозиторием](https://docs.rs.school/#/private-repository?id=Как-работать-с-приватным-репозиторием)
-- в приватном репозитории школы от ветки предыдущего задания создайте ветку с названием задания
-- история коммитов должна отображать процесс разработки приложения. [Требования к коммитам](https://docs.rs.school/#/git-convention?id=Требования-к-именам-коммитов)
-- после окончания разработки необходимо сделать Pull Request из ветки приложения в ветку предыдущего задания [Требования к Pull Request](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr). **Мержить Pull Request из ветки разработки в ветку предыдущего задания не нужно**
+- the task should be done in the school's **private repository** [How to work with a private repository](https://docs.rs.school/#/private-repository?id=Как-работать-с-приватным-репозиторием) [RU]
+- create a branch with the name of the task in the school's private repository from the branch of the previous task. 
+- The commit history should display the development process of the application. [Commit-name-requirements](https://www.conventionalcommits.org/en)
+- after the development is completed, it is necessary to make a Pull Request from the application branch to the branch of the previous task [Requirements for a Pull Request](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr) [RU]. **You do not need to merge a Pull Request from the development branch to the branch of the previous task**
```

Comment:
https://docs.app.rs.school/#/platform/pull-request-review-process
please change the link

Reply:
changed